common/djangoapps/student/management/commands/export_staff_users.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import absolute_import, print_function
+
+import logging
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.template.loader import get_template
+from os import remove
+
+from djqscsv import write_csv
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.models import CourseAccessRole
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    help = """
+    This command will export a csv of all users who have logged in within the given days and
+    have staff access role in active courses (Courses with end date in the future).
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('days', type=int, help='Indicate the login time period in days starting from today')
+
+    subject = 'Staff users CSV'
+    to_addresses = ['aazam@edx.org']
+    from_address = settings.DEFAULT_FROM_EMAIL
+    txt_template_path = 'email/export_staff_users.txt'
+    html_template_path = 'email/export_staff_users.html'
+    csv_filename = 'staff_users.csv'
+
+    def handle(self, *args, **kwargs):
+        days = kwargs['days']
+        current_date = datetime.now()
+        last_week_date = current_date - timedelta(days=days)
```

Comment:
Do not call it `last_week_date` because it could not. it depends upon the user's input or default value.